### PR TITLE
Login to github with oauth

### PR DESCRIPTION
**Pull request by bigfoodK**
Use github oauth web flow.
It only works with electron.
I have tested only in the wsl-windows-electron environment.

### CODE REVIEW

File: luda-editor/luda-editor-client/src/app/authentication/authentication.rs [hunk 1]
```diff
@@ -4,49 +4,57 @@ use namui_prebuilt::simple_rect;
 use std::sync::Arc;
 use wasm_bindgen_futures::spawn_local;
 
+const CLIENT_ID: &str = "Iv1.7b68d8d6f7b401df";
+const CLIENT_SECRET: &str = "350c026f799464cc1f0cb080325ba666d758e06d";
+
```
Comment (bigfoodK) [inline luda-editor/luda-editor-client/src/app/authentication/authentication.rs:9]: Client_secret should secret... but...

File: luda-editor/luda-editor-client/src/app/app.rs [hunk 2]
```diff
@@ -99,7 +103,8 @@ enum AppStage {
     },
 }
 
-fn initialize_app(github_api_client: Arc<GithubApiClient>) {
+fn initialize_app(_user_id: u32, github_api_client: Arc<GithubApiClient>) {
+    // TODO: use user_id
```
Comment (bigfoodK) [inline luda-editor/luda-editor-client/src/app/app.rs:107]: Someone will use this user_id.

**Comment by bigfoodK** [comment]
Uh...

**Comment by namse** [comment]
Why it merged?

**Comment by namse** [comment]
Ah I found the reason. Only master branch is protected with code reivew.

image

**Comment by bigfoodK** [comment]
It is not mandatory because we have already discussed enough.
However, the oauth client should be replaced. I added this to TODO of #286.

**Pull request merged**